pip/_vendor/html5lib/filters/alphabeticalattributes.py

```diff
@@ -5,7 +5,7 @@
 try:
     from collections import OrderedDict
 except ImportError:
-    from pip._vendor.ordereddict import OrderedDict
+    from ordereddict import OrderedDict
```

Comment:
Nah, this comes from html5lib and since we're no longer vendoring it then we leave the import alone.